### PR TITLE
A few cosmetic changes to the template namelists.

### PR DESCRIPTION
**Pull request by aubreyd**
No hurry on this PR...

### CODE REVIEW

File: trunk/NDHMS/template/NoahMP/namelist.hrldas [hunk 1]
```diff
@@ -1,72 +1,73 @@
...
+                   !1: use parallel io for reading multiple restart files (1 per core)
+rst_bi_out = 0     !0: use netcdf output restart file
+                   !1: use parallel io for outputting multiple restart files (1 per core)
 
- ! Restart file format options
- rst_bi_in = 0      !0: use netcdf input restart file
-                    !1: use parallel io for reading multiple restart files (1 per core)
- rst_bi_out = 0     !0: use netcdf output restart file
-                    !1: use parallel io for outputting multiple restart files (1 per core)
-/
```
Comment (kafitzgerald) [inline trunk/NDHMS/template/NoahMP/namelist.hrldas]: Did we want to get rid of this character?

File: trunk/NDHMS/template/NoahMP/namelist.hrldas [hunk 2]
```diff
@@ -1,72 +1,73 @@
...
 
- ! Specification of forcing data:  1=HRLDAS-hr format, 2=HRLDAS-min format, 3=WRF, 
- !                                 4=Idealized, 5=Ideal w/ spec. precip, 
- !                                 6=HRLDAS-hr format w/ spec. precip,
- !                                 7=WRF w/ spec. precip 
- FORC_TYP = 1
+! Specification of forcing data:  1=HRLDAS-hr format, 2=HRLDAS-min format, 3=WRF,
+!    4=Idealized, 5=Idealized w/ Spec. Precip.,
+!    6=HRLDAS-hourly fomat w/ Spec. Precip., 7=WRF w/ Spec. Precip.,
+!    9=Channel-only forcing, see hydro.namelist output_channelBucket_influxes
```
Comment (kafitzgerald) [inline trunk/NDHMS/template/NoahMP/namelist.hrldas]: Do we have an option 8?

Also, do we need disclaimers for option 9 and 10?  Maybe they're in the hydro.namelist?

File: trunk/NDHMS/template/NoahMP/namelist.hrldas (hunk 1 above)
Comment (rcabell) [reply trunk/NDHMS/template/NoahMP/namelist.hrldas]: is this just a whitespace change?

Comment (rcabell) [reply trunk/NDHMS/template/NoahMP/namelist.hrldas]: I think the backslash needs to be there... but that may be compiler-dependent

Comment (jmccreight) [reply trunk/NDHMS/template/NoahMP/namelist.hrldas]: The slash signifies the end of the namelist block.

File: trunk/NDHMS/template/NoahMP/namelist.hrldas (hunk 2 above)
Comment (jmccreight) [reply trunk/NDHMS/template/NoahMP/namelist.hrldas]: disclaimers? you mean "see hydro.namelist... "

**Comment by jmccreight** [comment]
Why is this a PR to v5.1.1? and not v5.1.0?

File: trunk/NDHMS/template/NoahMP/namelist.hrldas (hunk 2 above)
Comment (kafitzgerald) [reply trunk/NDHMS/template/NoahMP/namelist.hrldas]: I'm good w/ it if you are.

**Comment by jmccreight** [comment]
I think we also need to merge v5.1.1 into master? This PR is blocking that. So we either get this one merged soon or close it (temporarily) and make that merge, then reopen it.

**Comment by kafitzgerald** [comment]
> Why is this a PR to v5.1.1? and not v5.1.0?

Could be either.  Didn't think v5.1.0 was still evolving and this is pretty user centric.

**Comment by kafitzgerald** [comment]
> I think we also need to merge v5.1.1 into master? This PR is blocking that. So we either get this one merged soon or close it (temporarily) and make that merge, then reopen it.

I'll throw that character back in and then I think it should be mergeable.  If you'd rather wait though, I think that's fine too.

**Comment by jmccreight** [comment]
OK. Just wanted to be sure this isnt something desired in v5.1.0. There will be users that will use that verrsion, for sure. I'm fine with merging this to v5.1.1 and then merging that to master. Seems easiest.

File: trunk/NDHMS/template/NoahMP/namelist.hrldas (hunk 1 above)
Comment (aubreyd) [reply trunk/NDHMS/template/NoahMP/namelist.hrldas]: Ah yes, that was an accidental delete - thanks!

**Comment by aubreyd** [comment]
Was told to put into 5.1.1. I don't really care where it goes, just wanted it part of the eventual community release.

**Comment by rcabell** [comment]
The current thinking is that there will never be an official, supported v5.1.0 release; it will be v5.1.1 (which is currently out as a pre-release). Some users may use the older code, but at their own risk I guess.

**Pull request merged**